bot_cli/commands.py

```diff
@@ -264,3 +263,43 @@ def birthdays(args):
         print_success(result)
     else:
         print_warn(f"There is no one to celebrate birthday for next {period} day(s)")
+
+
+@contact_not_found_error
+@add_address_error
+def add_address(args):
```

Comment:
Docstring because it adds and updates address that isn't as obvious

Reply:
Doc added. Thanks.